open-api/rest-catalog-open-api.yaml

```diff
@@ -2068,6 +2165,145 @@ components:
           items:
             $ref: '#/components/schemas/PartitionStatisticsFile'
 
+    PlanTask:
+      description:
+        A flexible JSON object that contains information provided by the server,
+        to be utilized by clients for distributed planning. However it is recommended that servers
```

Comment:
I don't think we need to mention anything recommended. However, we should mention that it should be supplied as-is as input of PlanTable